README.md

```diff
@@ -197,6 +197,30 @@ describe Person do
 end
 ```
 
+### RSpec-Expectations
```

Comment:
This would be renamed to "Cucumber", and this whole section would talk about that instead of mentioning `rspec-expectations`.